themes/university.typ

```diff
@@ -0,0 +1,263 @@
+                        cell[
+                            #if "title" in slide-info {
+                              heading(level: 2, text(color-title)[#slide-info.title])
+                            }
+                        ],
+                        cell[
+                            #align(right)[
+                                #heading(level: 2, text(color-section)[#section.display()])
+                            ]
+                        ]
+                    )
+                ],
+            )
+        }
+
+        let footer-deco() = {
+            let cell = rect.with(
+                width: 100%,
+                inset: 1mm,
+                outset: 0mm
+            )
+            set text(
+                size: 10pt
+            )
+            set align(center)
+            grid(
+                columns: (25%, 1fr, 15%, 10%),
+                rows: (5mm, auto),
+                cell(
+                    height: 100%,
+                    fill: color-footer-fill-short-authors
+                )[#text(color-footer-short-authors)[#data.short-authors]],
+                cell(
+                    height: 100%,
+                    fill: color-footer-fill-short-title
+                )[#text(color-footer-short-title)[#data.short-title]],
+                cell(
+                    height: 100%,
+                    fill: color-footer-fill-date
+                )[#text(color-footer-date)[#data.date]],
+                cell(
+                    height: 100%,
+                    fill: color-footer-fill-slide-counter
+                )[#text(color-footer-slide-counter)[#logical-slide.display() / #last-slide-number]]
+            )
+        }
+
+        // header
+        header-deco()
+
+        v(1fr)
+        block(
+            width: 100%, inset: (x: 1em), breakable: false, outset: 0em,
+            body
+        )
+        v(2fr)
+
+        // footer
+        footer-deco()
+    }
+
+    let wake-up(slide-info, bodies) = {
+        if bodies.len() != 1 {
+            panic("wake-up variant of university theme only supports one body per slide")
+        }
+
+        let body = bodies.first()
+
+        block(
+            width: 100%,
+            height: 100%,
+            inset: 2em,
+            breakable: false,
+            outset: 0em,
+            fill: if "color" in slide-info {slide-info.color} else {black},
+            text(size: 2em, fill: white, {v(1fr); body; v(1fr)})
+        )
+    }
+
+    let split-v(slide-info, bodies) = {
+        if "columns" in slide-info {
+        } else {
+            slide-info.column = 2
+        }
+
+        let gridBox = box.with(
+            width: 100%,
+            height: 100%,
+            outset: 0em,
+            inset: (x: 0em),
+            baseline: 0em,
+            stroke: none
+        )
+
+        grid(
+            columns: slide-info.columns,
+            ..range(0, bodies.len()).map(i => gridBox(bodies.at(slide-info.columns - i - 1)))
```

Comment:
Why are you reversing the order of the bodies here?

Reply:
Fixed.